Milestone 1: Hello Microverse

RicardoEgas opened the pull request:
- Added HTML code;
- Added styling with CSS file;
- Implemented Linters to the project;
- Updated README file with descriptive information;
- Added MIT license;
- Add test.md file to .gitignore;

RicardoEgas committed:
Add linter file

RicardoEgas committed:
Add HTML file

RicardoEgas committed:
Add CSS file

RicardoEgas committed:
Add test.md file

RicardoEgas committed:
Ignore test.md file

RicardoEgas committed:
Added test.md file to gitignore

RicardoEgas committed:
Added *.md to .gitignore

RicardoEgas committed:
Added test.md to .gitignore

RicardoEgas committed:
Deleted test.md file

mukhammadrizooff commented on README.md:
```diff
@@ -1 +1,268 @@
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
- [ ] [REQUIRED] Good job so far! Please add `MIT` license because it isn't working 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/226639715-46939697-2ec2-48f0-a8e3-91d2a6d708e9.png)

mukhammadrizooff commented on README.md:
- [ ] [REQUIRED] Good job so far! Please provide descriptive information for all sections in the `README` 🙏🏼

mukhammadrizooff reviewed (changes requested):
Hi @RicardoEgas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Good job so far! Please add linters correctly because you forgot to add it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/226643227-436d3825-c634-4b6e-8321-58059a7bbb6b.png)


- [ ] [REQUIRED] Good job so far! `Pull Request` should have a `summary` describing introduced changes in more detail. Please add more details to your `PR` summary and edit the title of your pull request to a descriptive one🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/226641108-efea14d8-72c8-4741-be4b-18bba5261a3c.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

RicardoEgas committed:
Created .github/workflow folder

RicardoEgas committed:
Added linters config files

RicardoEgas committed:
Create LICENSE

RicardoEgas committed:
I added License and updated README file

RicardoEgas committed:
Merge branch 'feature1' of github.com:RicardoEgas/Hello_Microverse into feature1

RicardoEgas committed:
I updated linter file

RicardoEgas committed:
Deleted viewport scale content

RicardoEgas committed:
Deleted extra lines from index file

RicardoEgas committed:
Deleted extra lines from index file

RicardoEgas commented:
- Added HTML code;
- Added styling with CSS file;
- Implemented Linters to the project;
- Updated README file with descriptive information;
- Added MIT license;

aamirkhan2478 commented on README.md:
```diff
@@ -1 +1,268 @@
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
[OPTIONAL] I think the previous reviewer :point_up:  miss this portion so kindly remove `NOTE` from here because this is just for your information not for others.

aamirkhan2478 commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "16.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
```
- [ ] Great job :+1:  you added linters as suggested by the previous reviewer :point_up: but some linters are not passing kindly fix your all linters you can check this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) for more details.

aamirkhan2478 commented on README.md:
```diff
@@ -1 +1,268 @@
+**[Hello_Microverse]** is a project required for day 2 named "Set up a Hello Microverse project"
+
+## 🛠 Built With <a name="built-with"></a>
+
```
[OPTIONAL] This is optional but recommended please add your technologies like `HTML` and `CSS` here so the visitor will check which technologies you are working with. :slightly_smiling_face:

aamirkhan2478 commented on README.md:
```diff
@@ -1 +1,268 @@
+
+- **[HTML]**
+- **[CSS]**
+- **[Git]**
```
[OPTIONAL] Please add your `key features` here not your technologies you can add your technologies as I mentioned above :point_up: so kindly fix this issue also. :+1:

aamirkhan2478 reviewed (changes requested):
Hi @RicardoEgas , 

![](https://media3.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif?cid=ecf05e47cw5pjd1gapdzk1t3vp0n73caujoqxw3vkcu8ssng&rid=giphy.gif&ct=g)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.

 ## Required Changes ♻️
- [ ] Good job you edited your PR summary as suggested by the previous reviewer :point_up: but you didn't edit your PR title as suggested by the previous reviewer :point_up: please add a descriptive title like `Milestone 1: Hello Microverse`. :slightly_smiling_face: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] I think the previous reviewer :point_up:  miss this requirement your branch name is not descriptive your branch name should be a feature or action like `add-heading` please replace your branch name as I mentioned. :+1:   

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

RicardoEgas committed:
Added technologies to README file

RicardoEgas committed:
Deleted extra lines on index file

RicardoEgas committed:
Deleted extra file from DB

amon-cofie reviewed (approved):
Hi @RicardoEgas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights

🚀 Linters installed
🚀  Linters passing
🚀  Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
